src/Okta.Sdk.IntegrationTests/EventHooksClientShould.cs

```diff
@@ -0,0 +1,524 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    public class EventHooksClientShould
+    {
+        private const string SdkPrefix = "dotnet_sdk";
+        private const string EventType = "EVENT_TYPE";
+        
+        [Fact]
+        public async Task CreateEventHook()
+        {
+            var testClient = TestClient.Create();
+            var testEventHookName = $"{SdkPrefix}:{nameof(CreateEventHook)} Test Event Hook Name";
```

Comment:
If names have to be unique, then we have to append a random string here.

Reply:
Agreed, I was going to address that concern in a PR for work on OKTA-298370.